.github/workflows/ci.yml

```
@@ -468,6 +468,12 @@ jobs:
   python:
     runs-on: ubuntu-20.04
 
+    services:
+      rabbitmq:
+        image: rabbitmq
+        ports:
+          - 15672:15672
```

```suggestion
          - 5672:5672
```

si no, esto no va a corresponder con el puerto que se tiene en desarrollo local y es la razón por la cual no se puede conectar.

también falta hacer el `wait-for-it`. checa las demás llamadas en este mismo archivo.

---

Listo, a ver si está bien